pkg/kopia/cli/internal/args/args_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package args_test
+
+import (
+	"testing"
+
+	"github.com/kanisterio/kanister/pkg/kopia/cli"
+	"github.com/kanisterio/kanister/pkg/kopia/cli/internal/args"
+	"github.com/kanisterio/safecli/test"
+	"gopkg.in/check.v1"
```

Comment:
```suggestion
	"testing"

	"gopkg.in/check.v1"
	"github.com/kanisterio/safecli/test"

	"github.com/kanisterio/kanister/pkg/kopia/cli"
	"github.com/kanisterio/kanister/pkg/kopia/cli/internal/args"
```
Please place external packages to a separate group